core/Plugin/Controller.php

```diff
@@ -466,15 +466,18 @@ protected function getLastUnitGraphAcrossPlugins(
         }
 
         if (property_exists($view->config, 'selectable_columns')) {
-            $view->config->selectable_columns = array_merge($view->config->selectable_columns ? : array(), $selectableColumns);
+            $view->config->selectable_columns = array_merge($view->config->selectable_columns ?: array(), $selectableColumns);
         }
 
         $view->config->translations += $translations;
-
-        if ($reportDocumentation) {
-            $view->config->documentation = $reportDocumentation;
+    
+        // Add fallback for missing documentation
+        if (empty($reportDocumentation)) {
+            $reportDocumentation = Piwik::translate('General_NoDocumentationAvailable');
```

Comment:
The translation string doesn't exist.. so you may need to add it to the `lang/en.json` file.

Reply:
done! thanks again